install.sh

```diff
@@ -3599,9 +3590,7 @@ prepare_target_template_directory() {
         return 1
     fi
 
-    if [ "$USE_CORE_HOOKSPATH" = "yes" ]; then
-        set_githooks_directory "$TARGET_TEMPLATE_DIR"
-    fi
+    set_githooks_directory "$TARGET_TEMPLATE_DIR"
```

Comment:
Argh... this function made me go nuts...
Here thats the only single call to it, which makes understanding the code better.
First we find the template dir `$TARGET_TEMPLATE_DIR`:
- if  `--use-core-hookspath`  is set -> it will exist
-  if `--use-core-hookspath` not set -> its of the form `<dir>/hooks`, where <dir> exists, or  only `<dir>`